### ISSUE TITLE
Client tests leaving tmp files in /tmp

### ISSUE THREAD

**Issue by csams** [RedHatInsights/insights-core#1414]
Client tests create a dozen or so tmp files in `/tmp` that aren't cleaned up.

**Comment by Glutexo** [comment]
Do you refer to NamedTemporaryFiles created by the test suite?

**Comment by csams** [comment]
Yeah, there are several of those and then one for testing touch times.
